lessons/beginners/micropython/info.yml

```diff
@@ -0,0 +1,24 @@
+title: MicroPython
+style: md
+attribution: Pro PyLadies Brno napsal Petr Viktorin, 2016-2017.
```

Comment:
Oh dear; I don't remember if I took them or got them from China.
I took new ones today, and used those instead.